.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,34 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ develop, main, release/1.0.0 ]
+  pull_request:
+    branches: [ develop, release/1.0.0 ]
+  schedule:
+    - cron: '41 21 * * 2'
```

Comment:
At 21:41 on Tuesday? That's random 😄

Reply:
Yeah, their template picks time at random 😉